src/components/App.jsx

```diff
@@ -1,14 +1,80 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { hot } from 'react-hot-loader'
 
 import config from 'config/configFromCookie'
+import styles from 'styles/styles.scss'
 
 const { applicationPresentationName } = config
 
-// Just need to import this once somewhere so it ends up in the webpack dependency graph.
-// eslint-disable-next-line no-unused-vars
-import styles from 'styles/styles.scss'
+const LOCAL_API_URL = 'http://127.0.0.1:5000/mapping_data_frames/'
+const CLOUD_API_URL = 'http://ec2-54-144-23-157.compute-1.amazonaws.com/mapping_data_frames/';
+
+/*
+ * CreateMappingPage: React frontend for --create functionality.
+ *
+ * Prompts the user for a DBQ number, calls the corresponding mapping_data_frames 
+ * API endpoint, and downloads the resulting file.
+*/
+class CreateMappingPage extends Component {
+    constructor(props) {
+        /* Initializes app's dbqnum property and binds form event functions. */
+        super(props);
+        this.state = { dbqnum: '' };
```

Comment:
Let's add a state variable for loading. 

https://medium.com/swlh/the-nine-states-of-design-5bfe9b3d6d85